packages/manager/src/features/PlacementGroups/PlacementGroupsDetail/PlacementGroupsLinodes/PlacementGroupsLinodes.tsx

```diff
@@ -0,0 +1,104 @@
+import Grid from '@mui/material/Unstable_Grid2/Grid2';
+import * as React from 'react';
+
+import { Box } from 'src/components/Box';
+import { Button } from 'src/components/Button/Button';
+import { DebouncedSearchTextField } from 'src/components/DebouncedSearchTextField';
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
+import { Stack } from 'src/components/Stack';
+import { Typography } from 'src/components/Typography';
+import { useAllLinodesQuery } from 'src/queries/linodes/linodes';
+
+import {
+  MAX_NUMBER_OF_LINODES_IN_PLACEMENT_GROUP_MESSAGE,
+  PLACEMENT_GROUP_LINODES_ERROR_MESSAGE,
+} from '../../constants';
+import { hasPlacementGroupReachedCapacity } from '../../utils';
+import { PlacementGroupsLinodesTable } from './PlacementGroupsLinodesTable';
+
+import type { Linode, PlacementGroup } from '@linode/api-v4';
+
+interface Props {
+  placementGroup: PlacementGroup | undefined;
+}
+
+export const PlacementGroupsLinodes = (props: Props) => {
+  const { placementGroup } = props;
+  const {
+    data: allLinodes,
+    error: linodesError,
+    isLoading: linodesLoading,
+  } = useAllLinodesQuery();
```

Comment:
This isn't a big deal right now because sadly `useAllLinodesQuery` is used in a ton of places, but we might want to consider loading only the necessary Linodes using a filter. We could also paginate and search with the API filter if we want to be extra. 

```suggestion
  const {
    data: allLinodes,
    error: linodesError,
    isLoading: linodesLoading,
  } = useAllLinodesQuery(
    {},
    {
      '+or': placementGroup?.linode_ids.map((id) => ({
        id,
      })),
    }
  );
```